astropy/coordinates/tests/test_geodetic_representations.py

```diff
@@ -2,16 +2,65 @@
 
 """Test geodetic representations"""
 import pytest
-from numpy.testing import assert_array_equal
 
+from numpy.testing import assert_array_equal
```

Comment:
I think the numpy line should stay where it is, above a blank line with astropy imports.

Reply:
Indeed, sorry, I forgot to revert after some experimenting...